src/main/java/se/bjurr/violations/lib/example/OtherClass.java

```diff
@@ -4,11 +4,14 @@
  * No ending dot
```

Comment:
**Rule**: com.puppycrawl.tools.checkstyle.checks.metrics.BooleanExpressionComplexityCheck

Boolean expression complexity is 8 (max allowed is 1).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a154763392>*